packages/components/psammead-most-read/package.json

```diff
@@ -24,8 +24,8 @@
   ],
   "dependencies": {
     "@bbc/gel-foundations": "^3.4.3",
-    "@bbc/psammead-grid": "^1.1.5",
-    "@bbc/psammead-section-label": "^3.0.13",
+    "@bbc/psammead-grid": "^1.1.6",
+    "@bbc/psammead-section-label": "^4.0.0",
```

Comment:
This change might break the media indicator used in the Most Read, as it will require to add spacings if needed since we've removed them from the `Media Indicator`.

Reply:
Yeah I've just figured that, will close this